bentoml/cli/bento.py

```diff
@@ -260,3 +270,100 @@ def retrieve(bento, target_dir):
         safe_retrieve(bento_service_bundle_path, target_dir)
 
         click.echo('Service %s artifact directory => %s' % (name, target_dir))
+
+    @cli.command(
+        help='Containerize given Bento into a Docker image',
+        short_help="Containerize given Bento into a Docker image",
+    )
+    @click.argument("bento", type=click.STRING)
```

Comment:
We can utilize the callback option in Click.  It can be almost identical to the callback for bento as an option, you can find an example of it at https://github.com/bentoml/BentoML/blob/08defd2c461c614df8fe156c5b6c81aac354c969/bentoml/cli/azure_functions.py#L74